src/Poster/Poster.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { layoutOptions } from "./Poster.levers";
+import { Headline } from "./Poster.Headline";
+import { Content } from "./Poster.Content";
+import { Image } from "./Poster.Image";
+
+/**
+ * Poster Component
+ *
+ * HTML card element
+ */
+
+const PosterRoot = ({
+  bgImage: { src } = {},
+  bgGradient,
+  bgColor,
+  className,
+  children,
+  layout,
+}) => {
+  let wrapperClasses;
+  let imageWrapper;
+  let contentWrapper;
+
+  if (layout === "left") {
+    wrapperClasses = "su-flex su-flex-col su-justify-center md:su-flex-row";
+    imageWrapper =
+      "first:su-min-w-[14rem] su-rs-mb-2 su-mx-auto md:su-rs-mr-4 md:su-mb-0 md:su-ml-0";
```

Comment:
```suggestion
      "su-rs-mb-2 su-mx-auto md:su-rs-mr-4 md:su-mb-0 md:su-ml-0";
```

Sorry, no JIT compiler here.